src/actions/info.js

```diff
@@ -16,6 +16,10 @@ class ActionsInfo {
       this._store.pubKey = response.identity_pubkey;
       this._store.syncedToChain = response.synced_to_chain;
       this._store.blockHeight = response.block_height;
+      if (!response.synced_to_chain) {
```

Comment:
No longer call ComponentDidMount in sidebar.js. Added an if statement to getInfo that will continuously call getInfo until sync is complete. This allows for the updating of blockHeight and syncedToChain and rerendering of the sidebar to show current status.

Reply:
Great. Any reason you chose 1000ms and not the `RETRY_DELAY` as used in the catch block?

Reply:
No, not really, I can change it.